apps/pie-storybook/stories/pie-toast-provider.stories.ts

```diff
@@ -25,10 +45,85 @@ const toastProviderStoryMeta: ToastProviderStoryMeta = {
 
 export default toastProviderStoryMeta;
 
-// TODO: remove the eslint-disable rule when props are added
-// eslint-disable-next-line no-empty-pattern
-const Template = ({}: ToastProviderProps) => html`
-    <pie-toast-provider></pie-toast-provider>
+const Template = ({ options }: ToastProviderProps) => {
+    const onQueueUpdate = (event: CustomEvent) => {
+        const queueLength = document.querySelector('#queue-length-tag') as HTMLElement;
+        if (queueLength) {
+            queueLength.textContent = `Toast Queue Length: ${event.detail.length}`;
+        }
+    };
+
+    return html`
+    <pie-toast-provider 
+        .options=${options} 
+        @pie-toast-provider-queue-update=${onQueueUpdate}>
+    </pie-toast-provider>
+
+    <pie-tag id="queue-length-tag" variant="information" style="margin-top: 16px;">
+        Toast Queue Length: 0
+    </pie-tag>
+
+    <div style="margin-top: 16px; display: flex; gap: 16px; flex-wrap: wrap;">
+        <pie-button 
+            @click=${() => {
+        toaster.create({
+            message: 'Low Priority Info',
+            variant: 'info',
+        });
```

Comment:
Would it be ok to move the indentation of these buttons click handlers content?

```suggestion
        <pie-button 
            @click=${() => {
              toaster.create({
                  message: 'Low Priority Info',
                  variant: 'info',
              });
```

Reply:
it's auto-formatted on save for me 🙄 are you getting the same thing locally? 

https://github.com/user-attachments/assets/9c34bdb3-4013-474b-8017-0a49ee6ea25a